bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -424,6 +426,66 @@ impl OmniConnector {
         Ok(outcome)
     }
 
+    pub async fn bind_token_eth_evm_prover(
```

Comment:
Not sure I understand the naming of this method

Reply:
fixed: https://github.com/Near-One/rainbow-bridge-sdk-rs/pull/37/commits/18fb13334bb0ada4358063145d7ae121efd9d5b8